src/zimit/zimit.py

```diff
@@ -558,6 +558,13 @@ def get_node_cmd_line(args):
         "logging",
     ]:
         value = getattr(args, arg)
+        if arg == "userAgent":
+            # strip leading whitespace which are not allowed on some websites
+            if value:
```

Comment:
Why this test? This can only be an str no? Why not directly stripping?

Reply:
value is None when userAgent is not passed at the CLI; comment updated to make it more obvious (I had the same feeling at first as well)